docs/README.md

```diff
@@ -1,43 +1,78 @@
+Hi! 👋 Welcome to the Block Editor Handbook.
+
+The **block editor** is a modern and up-to-date paradigm for WordPress site building and publishing.
+
+It uses a modular system of **Blocks** to compose and format content, and is designed to create rich and flexible layouts for websites and digital products.
+
+The editor consists of a number of key elements which are highlighted in the following graphic:
 
 ![Quick view of the block editor](https://raw.githubusercontent.com/WordPress/gutenberg/trunk/docs/assets/quick-view-of-the-block-editor.png)
 
 **Legend:**
 
-1. Block inserter
-2. Block editor content area
-3. Settings sidebar
+1. _Block inserter_
+2. _Block editor content area_
+3. _Settings sidebar_
+
+Content is created modularly using discrete blocks. A block is essentially a wrapper that contains and styles one or more individual items of content. The block editor is in contrast to the older TinyMCE editor which used freeform text with inserted media, embeds and Shortcodes. The block editor enables a superior editing experience and is intended to replace the TinyMCE editor.
+
+A block is a discrete element such as a Paragraph, a Heading, a Media element, or an Embed. Each element is treated as a separate component. When all these components are pieced together, they make up the content of a post or page that is then stored in the WordPress database.
+
+The block editor is designed with progressive enhancement in mind, meaning that it is backward compatible with all legacy content. Blocks also offer enhanced editing and format controls.
+
+Not only does the block editor enhance the editing experience by offering visual drag-and-drop content creation tools, it is also a powerful developer platform with a rich feature-set of APIs that allow it to be manipulated and extended in a multitude of different ways.
+
+### The Gutenberg project
 
-Using a system of Blocks to compose and format content, the new block-based editor is designed to create rich, flexible layouts for websites and digital products. Content is created using blocks instead of freeform text with inserted media, embeds and Shortcodes (there's a Shortcode block, though).
+“[Gutenberg](https://github.com/WordPress/gutenberg/)” is the codename for the project to develop the block editor. It aims to revolutionize the entire publishing experience, just as Johannes Gutenberg revolutionized the printed word back in the 15th century. 
```

Comment:
`Publishing experience` seems out of place in the context of the editor. It is the only reference to publishing in this document and doesn't fit. Maybe "the WordPress editing experience"?